src/view/configuration/configuration.jsx

```diff
@@ -227,11 +227,11 @@ const validationSchema = object()
   // TestCafe doesn't allow this to be an arrow function because of
   // how it scopes "this".
   // eslint-disable-next-line func-names
-  .test("uniqueImsOrgId", function(settings) {
+  .test("uniqueorgId", function(settings) {
```

Comment:
I don't remember how the name actually provides value here, but I think we should camelcase it (`uniqueOrgId`).

Reply:
Will do, and will update `runAlloy.js`. Thanks!